src/css/components/_button.scss

```diff
@@ -13,6 +13,7 @@
 
 .button--blue {
   background: var(--blue);
+  margin-bottom: 10px;
```

Comment:
Has this been pulled in from the previous PR?

Reply:
Oh yes, I should've created a new branch from the master branch instead of "gap branch".